pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -51,7 +51,7 @@ message MessageMetadata {
 	optional CompressionType compression = 8 [default = NONE];
 	optional uint32 uncompressed_size = 9 [default = 0];
 	// XXHash64 checksum of the original message payload
-	optional sfixed64 checksum = 10;
+	//optional sfixed64 checksum = 10;
```

Comment:
Add comment to mention this field was removed in favor of header+payload checksum